### DISCUSSION TITLE
Bring an apk or obb version

### DISCUSSION THREAD

**Discussion by opticSquid** [nukeop/nuclear#1164] [Ideas]
Please bring out an apk or Obb version of this awesome product

**Comment by nukeop** [comment]
https://nukeop.gitbook.io/nuclear/faq#is-there-an-android-ios-version

**Comment by ujlaki15** [comment]
The .apk version would be of use for me too. But I've read the information you provided under the link. Nevermind it. I am speechless already, how great you guys are. I can live with the fact I may never be able to use Nuclear on my smartphone. Just let me give you my warmest appreciation.

**Comment by riccardosimo** [comment]
any news about an android version? :(
